src/client/components/icons/BackArrow.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react'
 
 export const BackArrow = () => (
-  <svg xmlns="http://www.w3.org/2000/svg" width="11" height="18" fill="#9B9BAA">
-    <path d="M9.895 2.79a.8.8 0 000-1.139l-.562-.554a.8.8 0 00-1.124 0L.772 8.43a.8.8 0 000 1.14l7.437 7.333a.8.8 0 001.124 0l.562-.554a.8.8 0 000-1.14L3.597 9l6.298-6.21z" />
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M9.06066 12L16.5303 4.53033L15.4697 3.46967L6.93934 12L15.4697 20.5303L16.5303 19.4697L9.06066 12Z"
+      fill="#777777"
```

Comment:
This value can be imported from `@hedviginsurance/brand` package :)

Reply:
I don't think we have a color in the brand package that is the same, should I just go with the closest one or is there another way around that?

Reply:
Maybe ask Zak if we should update brand colors or if you can use a color that's in the brand package? Because he's the one who gave you these icons right?

Reply:
I checked with Zak and he said it was just a placeholder for us to override with whatever color is supposed to be there. I checked how we have done with other Icons and for many of them we have just left fill out completely or set it to "none" in the icon components themselves so I did that now. The color is set then where the icon is used... Or should we rethink that?

Reply:
Exactly @amandaflod ,colors should be set where they are used 🙌 In the case of this color, @robinandeer added `gray600: '#777777'` a few days ago since Zak wanted another tint. So now we have the full scale of grays (in Brand repo). 
```
  gray100: '#fafafa',
  gray200: '#f2f2f2',
  gray300: '#eaeaea',
  gray500: '#aaaaaa',
  gray600: '#777777',
  gray700: '#505050',
  gray800: '#272727',
  gray900: '#121212',
```